### PR TITLE
Salma renamed 2 columns in categories_projects

### PR DESCRIPTION
**Pull request by salmafg**
@m0hamed

### CODE REVIEW

File: arability/db/migrate/20130327210228_categories_projects.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CategoriesProjects < ActiveRecord::Migration
+  def up
+  	drop_table :categories_projects
+  	create_table :categories_projects do |t|
+  	t.references :category
+  	t.references :project
+  end
```
Comment (m0hamed) [inline arability/db/migrate/20130327210228_categories_projects.rb]: please fix indentaion

**Comment by m0hamed** [comment]
there are three migrations doing the exact same thing please remove the first 2 and leave the third

File: arability/db/migrate/20130327210228_categories_projects.rb (hunk 1 above)
Comment (salmafg) [inline arability/db/migrate/20130327210228_categories_projects.rb]: This one's deleted

Comment (salmafg) [inline arability/db/migrate/20130327210228_categories_projects.rb]: Mana b2ollk 2 of the 3 files changed are deleted.

Comment (m0hamed) [inline arability/db/migrate/20130327210228_categories_projects.rb]: lazem te3mely lel deletion commit we ba3deen push

Comment (salmafg) [inline arability/db/migrate/20130327210228_categories_projects.rb]: Omal ana 3amalt eh?

Comment (m0hamed) [inline arability/db/migrate/20130327210228_categories_projects.rb]: you deleted them but didnt commit the delete. when you do git status you
will see they have been deleted but not commited

On Fri, Mar 29, 2013 at 1:56 PM, Salma Farag Galal <notifications@github.com

> wrote:
> 
> In arability/db/migrate/20130327210228_categories_projects.rb:
> 
> > @@ -0,0 +1,12 @@
> > +class CategoriesProjects < ActiveRecord::Migration
> > -  def up
> > -   drop_table :categories_projects
> > -   create_table :categories_projects do |t|
> > -   t.references :category
> > -   t.references :project
> > -  end
> 
> Omal ana 3amalt eh?
> … <#13db601d6294922d_>

Comment (salmafg) [inline arability/db/migrate/20130327210228_categories_projects.rb]: How do I commit deleted files? Ana 3ammala agarrab 7agat bs msh fahma

Comment (m0hamed) [inline arability/db/migrate/20130327210228_categories_projects.rb]: git rm <path to file> then git commit

On Fri, Mar 29, 2013 at 2:14 PM, Salma Farag Galal <notifications@github.com

> wrote:
> 
> In arability/db/migrate/20130327210228_categories_projects.rb:
> 
> > @@ -0,0 +1,12 @@
> > +class CategoriesProjects < ActiveRecord::Migration
> > -  def up
> > -   drop_table :categories_projects
> > -   create_table :categories_projects do |t|
> > -   t.references :category
> > -   t.references :project
> > -  end
> 
> How do I commit deleted files? Ana 3ammala agarrab 7agat bs msh fahma
> … <#13db611dfd8a006e_>

Comment (salmafg) [inline arability/db/migrate/20130327210228_categories_projects.rb]: By2olly did not match any file

Comment (salmafg) [inline arability/db/migrate/20130327210228_categories_projects.rb]: Ashraaf I committed wa7ed mnhom el tany msh shyfu

**Comment by m0hamed** [comment]
there is still an extra migration.

**Comment by salmafg** [comment]
Check again now.

**Comment by m0hamed** [comment]
approved

**Pull request merged**